### PR TITLE
[doc] faq How to diagnose job problems through logs

### PR DESCRIPTION
**Pull request by YanjieGao**
fan's suggestion to add a tutorial for user about how to diagnose job problems through logs

### CODE REVIEW

**Comment by qyyy** [comment]
Is it better to list the common reasons of job failure and job retry?

**Comment by YanjieGao** [comment]
just saw failure and retry types at framework launcher user manual, i will add a link to these part.

**Comment by qyyy** [comment]
What's more, I think it's necessary to note that yarn can only store 1000 jobs' log. So, if the users want to find some old logs, like the frequent retry jobs, he or she maybe cannot find it.

**Comment by YanjieGao** [comment]
good suggestion

**Comment by yqwang-ms** [comment]
Pls refer this:
[Diagnose LauncherAM](https://microsoft.sharepoint.com/teams/OpenAIInfra/_layouts/OneNote.aspx?id=%2Fteams%2FOpenAIInfra%2FSiteAssets%2FOpenAIInfra%20Notebook&wd=target%28DRI.one%7CD2745523-A19B-462A-956D-89DE5198BFAF%2FDiagnose%20LauncherAM%7C9352D905-C01B-4142-8CE5-0325FBB6FE77%2F%29)

[Diagnose LauncherService/LauncherWebServer](https://microsoft.sharepoint.com/teams/OpenAIInfra/_layouts/OneNote.aspx?id=%2Fteams%2FOpenAIInfra%2FSiteAssets%2FOpenAIInfra%20Notebook&wd=target%28DRI.one%7CD2745523-A19B-462A-956D-89DE5198BFAF%2FDiagnose%20LauncherService%5C%2FLauncherWebServer%7CBDB6B2E3-70CF-46AC-AB6D-7D0EC54F800C%2F%29)

[Diagnose Launcher Framework (Job)](https://microsoft.sharepoint.com/teams/OpenAIInfra/_layouts/OneNote.aspx?id=%2Fteams%2FOpenAIInfra%2FSiteAssets%2FOpenAIInfra%20Notebook&wd=target%28DRI.one%7CD2745523-A19B-462A-956D-89DE5198BFAF%2FDiagnose%20Launcher%20Framework%20%28Job%5C%29%7C8853709C-C8BB-47F0-9BC8-A508791299F8%2F%29)

**Comment by YanjieGao** [comment]
thanks yuqi, this is mainly referred [Diagnose Launcher Framework (Job)](https://microsoft.sharepoint.com/teams/OpenAIInfra/_layouts/OneNote.aspx?id=%2Fteams%2FOpenAIInfra%2FSiteAssets%2FOpenAIInfra%20Notebook&wd=target%28DRI.one%7CD2745523-A19B-462A-956D-89DE5198BFAF%2FDiagnose%20Launcher%20Framework%20%28Job%5C%29%7C8853709C-C8BB-47F0-9BC8-A508791299F8%2F%29)

File: docs/job_log.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+we could get information:
+1.  UserApplication itself failed.
+2.  cntk-test-4621-17223-container_e9878_1532412068340_0018_01_000002 does not exist is the reason.
+3.  Then we could visit ```http://10.151.40.165:8042/node/containerlogs/container_e9878_1532412068340_0018_01_000002/core/``` at step 1.2 to detect task failure reason.
+
+
+### 1.2 View job each task container log  <a name="tasklog"></a>
+
+- Check the failed task log who triggered the whole attempt failed, i.e.
```
Comment (fanyangCS) [inline docs/job_log.md]: >Check [](start = 2, length = 5)

Please add the information about log rotation, and refer to the corresponding configuration file (or doc if we have)

Comment (YanjieGao) [reply docs/job_log.md]: Hi @ydye, currently, The default configuration of log rotation is at which file ? user could configure which items to control log rotation?

**Pull request merged**